### PR TITLE
Clarify that the Microsoft Teams PWA have more features that the normal Desktop App

### PR DESCRIPTION
**Pull request by traversaro**

### CODE REVIEW

File: doc/microsoft-teams-screen-share-ubuntu-22-04.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ choose **only one** of the following two solutions:
 * S1: Use the web version of Microsoft Teams with Chromium
 * S2: Disable Wayland and use Xorg
 
+Between the two solution, S1 is preferred as the web version of Microsoft Teams, beside supporting screen sharing on Wayland, also supports features that are not supported in the Microsoft Teams app you can get from https://www.microsoft.com/en-us/microsoft-teams/download-app , such as having more than four persons in the video when you are having a video chat.
```
Comment (HosameldinMohamed) [inline doc/microsoft-teams-screen-share-ubuntu-22-04.md:10]: Let's verify the feature "showing more than 4 participants first". I vaguely remember it was possible, but let's double-check it.

In theory, it's mentioned in this post

https://techcommunity.microsoft.com/t5/microsoft-teams-blog/microsoft-teams-progressive-web-app-now-available-on-linux/bc-p/3671394/highlight/true

> The PWA offers access to more capabilities, including custom backgrounds, gallery view, reactions, the raise-a-hand feature in meetings, as well as large gallery and Together mode views. The PWA also provides desktop-like app features, such as system notifications for chat and channel, a dock icon with respective controls, application auto-start, and easy access to system app permissions.

**Comment by traversaro** [comment]
@HosameldinMohamed do you think this is more clear?

**Comment by HosameldinMohamed** [comment]
> @HosameldinMohamed do you think this is more clear?

No no what you wrote is ok. I just got confused by the post because I didn't understand what they mean by `gallery view` and `large gallery view`, but they are explained here https://support.microsoft.com/en-us/office/customize-your-meeting-view-95aaeaf8-0f22-46cf-a6f9-34ca9b04a1b2

**Review by HosameldinMohamed — APPROVED** [review]
Good for me!

**Pull request merged**